compliance-monitor/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10-alpine
+WORKDIR /code
+COPY . .
+RUN \
+ apk add --no-cache postgresql-libs && \
+ apk add --no-cache --virtual .build-deps gcc musl-dev postgresql-dev && \
+ python3 -m pip install -r requirements.txt --no-cache-dir && \
+ apk --purge del .build-deps
+EXPOSE 8080
+CMD ["python3", "./monitor.py"]
```

Comment:
A good pattern might also be the creation of a user and the execution of the server with that user.

```
RUN useradd -g users  -u 1001 -m -s /bin/bash runuser
USER 1001
....
WORKDIR /
```